.github/workflows/daily.yml

```diff
@@ -0,0 +1,26 @@
+name: Daily updates for v2
+
+# For testing
+on: push
+
+# For prod
+# on:
+#   schedule:
+#     - cron: '0 4 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+      - run: npm install
+      - run: npm run build
+
+      # Go through all open PRs and run the bot over them
+      - run: node ./bin/scripts/daily.js
+        env:
+          BOT_AUTH_TOKEN: ${{ secrets.BOT_AUTH_TOKEN }}
+
+   
```

Comment:
Right now this is set to run when it's merged, so I can test it, then I'll push to master switching it over to the cron